### PR TITLE
Fix ShardClientUtil#broadcastEval - now really accepting functions

### PR DESCRIPTION
**Pull request by pedall**
**Please describe the changes this PR makes and why it should be merged:**
There is a small typo in the broadcastEval method. 

It should be 
```
script = typeof script === 'function' ? `(${script})(this)` : script;
``` 
instead of 
```
script = typeof script !== 'function' ? `(${script})(this)` : script;
```

Additionally adding `.idea` to the .gitignore file, so that also jetbrain products like Webstorm and Idea are ignored (additionally to vscode)

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by appellation** [comment]
without testing, it seems like the current implementation works as intended: ie. if the script is a function, wrap it and pass the client.

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ deploy/deploy_key.pub
 .vscode/
 docs/docs.json
 webpack/
+.idea/
```
Comment (SpaceEEC) [inline .gitignore]: Is this relevant to the PR itself? 👀

Comment (pedall) [reply .gitignore]: No but I thought since .vscode is already there why not also help people using webstorm as IDE to not have project files alles each time :eyes:

Comment (pedall) [reply .gitignore]: added*

Comment (iCrawl) [reply .gitignore]: Because it really should be in a seperate PR.

**Comment by pedall** [comment]
removed `.idea` from .gitignore

**Comment by iCrawl** [comment]
Beautiful, thank you.
